Community-Supported/native-s3/README.md

```diff
@@ -69,4 +69,4 @@ Check out these resources to learn more:
 
 - [The EXTERNAL function in the Hyper API SQL Reference](https://help.tableau.com/current/api/hyper_api/en-us/reference/sql/functions-srf.html#FUNCTIONS-SRF-EXTERNAL)
 
-- [AWS command line tools documentation](https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html), e.g. if you want to download some of the sample files to your local machine and explore them
\ No newline at end of file
+- [AWS command line tools documentation](https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html), e.g. if you want to download some of the sample files to your local machine and explore them
```

Comment:
what changed here?

I would rather leave this file unchanged...

Reply:
I would leave this point in as it tells ppl how to download files from s3 with the amazon cli to explore.

Reply:
@jonas-eckhardt not sure I understand your reply.

I think we both agree that this line should stay. But my question is: Why was it even touched in this commit? What changed here?

Reply:
Sorry I did not want to change anything there, just wanted to revert to the previous version